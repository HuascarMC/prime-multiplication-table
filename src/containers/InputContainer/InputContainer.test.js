import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

xit('should render without crashing', () => {
 shallow(<InputContainer />);
})

xit('should render render one form', () => {
 const wrapper = shallow(<InputContainer />);
 expect(wrapper.find('.get-number-form').to.have.length(1));
})

xit('should render input box', () => {
 const wrapper = shallow(<InputContainer />);
 expect(wrapper.find('.get-number-input').to.have.length(1));
})

xit('should render text displaying input', () => {
 const wrapper = shallow(<InputContainer />);
 expect(wrapper.find('.display-input-text').to.equal(0))
})

xit('should have state number 0', () => {
 const wrapper = shallow(<InputContainer />);
 expect(wrapper.state().number).to.equal(0);
})

xit('it should update state number', () => {
 const wrapper = shallow(<InputContainer />);
 wrapper.find('.get-number-input').simulate('click').simulateKeyPresses(9);
 expect(wrapper.state().number).to.equal(9);
})
