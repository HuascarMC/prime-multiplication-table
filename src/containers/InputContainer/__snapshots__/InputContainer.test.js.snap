// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<form
  className="get-number-form"
>
  <p
    className="display-input-text"
  >
    0
  </p>
  <input
    className="get-number-input"
    onChange={[Function]}
  />
  <table
    className="primes-table"
  >
    <tbody>
      <tr />
    </tbody>
  </table>
</form>
`;
